requirements/base.txt

```diff
@@ -50,15 +48,12 @@ six==1.16.0
     # via
     #   -r requirements/base.in
     #   social-auth-app-django
-    #   social-auth-core
 social-auth-app-django==4.0.0
     # via
     #   -c requirements/common_constraints.txt
     #   -r requirements/base.in
-social-auth-core==4.0.2
+social-auth-core==4.1.0
```

Comment:
`social-auth-core 4.0.2 requires PyJWT<2,>=1.4.0, but you have pyjwt 2.1.0 which is incompatible`